eiffel-syntax-and-usage/derived-confidence-level.md

```diff
@@ -0,0 +1,34 @@
+# Derived Confidence Level
+
+As mentioned in EiffelConfidenceLevelModifiedEvent, the EiffelConfidenceLevelModifiedEvent declares that an entity has achieved (or failed to achieve) a certain level of confidence, or in a broader sense to annotate it as being applicable or relevant to a certain case (e.g. fit for release to a certain customer segment or having passed certain criteria.
```

Comment:
Make the event name a link to relevant documentation.
Missing closing parenthesis at end of paragraph.